### PR TITLE
[Commands] Cleanup #reloadzps Command.

### PR DESCRIPTION
**Pull request by Kinglykrab**
- Cleanup messages and logic.
- Make reloading of zone points global instead of zone specific.
- Add zone->GetZoneDescription().
- Add mob->GetTargetDescription(mob).
- Cleanup all zoneserver packets and make use of auto.
- Fix all spots where GetTargetDescription()/GetZoneDescription() would fit.

### CODE REVIEW

File: zone/worldserver.cpp [hunk 1]
```diff
@@ -2013,6 +2013,34 @@ void WorldServer::HandleMessage(uint16 opcode, const EQ::Net::Packet &p)
...
+	case ServerOP_ReloadZonePoints: {
+		if (zone) {
+			worldserver.SendEmoteMessage(
+				0,
+				0,
+				AccountStatus::GMAdmin,
+				Chat::Yellow,
+				fmt::format(
+					"Zone points reloaded for {}{}.",
+					fmt::format(
```
Comment (Akkadius) [inline zone/worldserver.cpp:2025]: Similar recommendation as in other PR; `zone->DescribeZoneLabel()`

**Review by Akkadius — APPROVED** [review]

File: zone/worldserver.cpp (hunk 1 above)
Comment (Kinglykrab) [reply zone/worldserver.cpp:2025]: 👍

File: zone/gm_commands/task.cpp [hunk 2]
```diff
@@ -359,21 +326,12 @@ void command_task(Client *c, const Seperator *sep)
...
-							client_target->GetCleanName(),
-							client_target->GetID()
-						)
-					)
+					c->GetTargetDescription(target)
 				).c_str()
 			);
 
-			client_target->UpdateTaskActivity(task_id, activity_id, count);
-			c->ShowClientTasks(client_target);
```
Comment (Akkadius) [inline zone/gm_commands/task.cpp:375]: Why is this flipped

File: zone/gm_commands/task.cpp [hunk 3]
```diff
@@ -282,62 +266,45 @@ void command_task(Client *c, const Seperator *sep)
 
 		return;
 	} else if (is_show) {
-		c->ShowClientTasks(client_target);
+		target->ShowClientTasks(c);
 		return;
 	} else if (is_uncomplete) {
 		if (sep->IsNumber(2)) {
 			auto task_id = std::stoul(sep->arg[2]);
-			if (task_id && task_id < MAXTASKS) {
```
Comment (Akkadius) [inline zone/gm_commands/task.cpp:290]: Not sure why this changed

**Comment by Akkadius** [comment]
There are a ton of changes in this PR that happened after approval. Not ideal especially when it can be the source of issues, it can be hard to weed out the problem

File: zone/gm_commands/task.cpp (hunk 3 above)
Comment (Kinglykrab) [reply zone/gm_commands/task.cpp:290]: The logic flow changed, the return comes first so we fail out instead of it coming after.

**Pull request merged**